src/OrchardCore.Modules/OrchardCore.Users/Assets/js/password-generator.js

```diff
@@ -0,0 +1,64 @@
+passwordManager = function () {
+    const allowed = {
+        uppers: 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',
+        lowers: 'abcdefghijklmnopqrstuvwxyz',
+        digits: '0123456789',
+        symbols: '!#$%&\()*+,-./:;<=>\'?@@[\]^_`{|}~'
```

Comment:
Are you sure all of these symbols allowed by Identity `PasswordGenerator`?

Reply:
yes. Identity does not care what special characters you pass. a special character if defined if (not digit + not number)

Reply:
@hishamco more on that in this comment https://github.com/OrchardCMS/OrchardCore/pull/12024#discussion_r928135410